poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
```

Comment:
Maybe not an issue, but why is Poetry downgraded?

Reply:
@tilen1976 that just depends which version of Poetry we have installed locally, it's normal that it differs slightly